cot/src/handler.rs

```diff
@@ -105,79 +107,101 @@ macro_rules! impl_request_handler_from_request {
-impl_request_handler_from_request!(P1, P2, P3, P4, (P5));
-impl_request_handler_from_request!((P1), P2, P3, P4, P5, P6);
-impl_request_handler_from_request!(P1, (P2), P3, P4, P5, P6);
-impl_request_handler_from_request!(P1, P2, (P3), P4, P5, P6);
-impl_request_handler_from_request!(P1, P2, P3, (P4), P5, P6);
-impl_request_handler_from_request!(P1, P2, P3, P4, (P5), P6);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, (P6));
-impl_request_handler_from_request!((P1), P2, P3, P4, P5, P6, P7);
-impl_request_handler_from_request!(P1, (P2), P3, P4, P5, P6, P7);
-impl_request_handler_from_request!(P1, P2, (P3), P4, P5, P6, P7);
-impl_request_handler_from_request!(P1, P2, P3, (P4), P5, P6, P7);
-impl_request_handler_from_request!(P1, P2, P3, P4, (P5), P6, P7);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, (P6), P7);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, (P7));
-impl_request_handler_from_request!((P1), P2, P3, P4, P5, P6, P7, P8);
-impl_request_handler_from_request!(P1, (P2), P3, P4, P5, P6, P7, P8);
-impl_request_handler_from_request!(P1, P2, (P3), P4, P5, P6, P7, P8);
-impl_request_handler_from_request!(P1, P2, P3, (P4), P5, P6, P7, P8);
-impl_request_handler_from_request!(P1, P2, P3, P4, (P5), P6, P7, P8);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, (P6), P7, P8);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, (P7), P8);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, P7, (P8));
-impl_request_handler_from_request!((P1), P2, P3, P4, P5, P6, P7, P8, P9);
-impl_request_handler_from_request!(P1, (P2), P3, P4, P5, P6, P7, P8, P9);
-impl_request_handler_from_request!(P1, P2, (P3), P4, P5, P6, P7, P8, P9);
-impl_request_handler_from_request!(P1, P2, P3, (P4), P5, P6, P7, P8, P9);
-impl_request_handler_from_request!(P1, P2, P3, P4, (P5), P6, P7, P8, P9);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, (P6), P7, P8, P9);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, (P7), P8, P9);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, P7, (P8), P9);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, P7, P8, (P9));
-impl_request_handler_from_request!((P1), P2, P3, P4, P5, P6, P7, P8, P9, P10);
-impl_request_handler_from_request!(P1, (P2), P3, P4, P5, P6, P7, P8, P9, P10);
-impl_request_handler_from_request!(P1, P2, (P3), P4, P5, P6, P7, P8, P9, P10);
-impl_request_handler_from_request!(P1, P2, P3, (P4), P5, P6, P7, P8, P9, P10);
-impl_request_handler_from_request!(P1, P2, P3, P4, (P5), P6, P7, P8, P9, P10);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, (P6), P7, P8, P9, P10);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, (P7), P8, P9, P10);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, P7, (P8), P9, P10);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, P7, P8, (P9), P10);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, P7, P8, P9, (P10));
+#[rustfmt::skip]
```

Comment:
Why do we need `#[rustfmt::skip]` for this? (Same question for the next macro)

Reply:
We don't, copy-paste error :sweat_smile: